test/common/performance/aggregator/aggregator.go

```diff
@@ -49,7 +54,32 @@ type eventsRecord struct {
 	*pb.EventsRecord
 }
 
-var fatalf = log.Fatalf
+var (
+	fatalf = log.Fatalf
+
+	pea = &tpb.ThresholdAnalyzerInput{
```

Comment:
How solid do we think these values are? It might be nice to be able to change these without having to modify the code?

Reply:
We should always make sure the error rate is 0. In Serving, we are also hardcoding the CrossRunCount as [10](https://github.com/knative/serving/blob/25658ef8c7b1bee4e0996566833e2ced22cdef9b/test/performance/benchmarks/dataplane-probe/continuous/sla.go#L43) so the analyzer will fail if the median value of last 10 runs are larger than 0, and it works pretty well.

So I would prefer to keep it as it is, and make it configurable if we do need it in the future.